src/ParseServer.js

```diff
@@ -210,7 +211,16 @@ class ParseServer {
 
     Config.validate(AppCache.get(appId));
     this.config = AppCache.get(appId);
-    hooksController.load();
+    if (hooksRefreshIntervalMs > 0) {
+      (function repeat() {
+        setTimeout(() => {
```

Comment:
Can you just use cache adapter? That would solve this problem nicely, then we can also remove that extra server setting in favor of the existing setting.